docs/source/markdown/podman-volume-import.1.md

```diff
@@ -0,0 +1,32 @@
+% podman-volume-import(1)
+
+## NAME
+podman\-volume\-import - Import tarball contents into a podman volume
+
+## SYNOPSIS
+**podman volume import** [*options*] *volume* [*source*]
+
+## DESCRIPTION
+
+**podman volume import** imports the contents of a tarball into the podman volume's mountpoint.
+**podman volume import** can consume piped input when using `-` as source path. 
+
+Note: Following command is not supported by podman-remote.
+
+**podman volume import [OPTIONS] VOLUME [SOURCE]**
+
+#### **--help**
+
+Print usage statement
+
+## EXAMPLES
+
+```
+$ cat hello.tar | podman volume import myvol -
```

Comment:
Use an example with a compressed file

gunzip hellow.tar.gz 

That is a more valid use case.

Reply:
this is resolved in latest commit.